cpp/src/plasma/common/common_extension.cc

```diff
@@ -0,0 +1,292 @@
+#include <Python.h>
+#include "bytesobject.h"
+#include "node.h"
+
+#include "common.h"
+#include "common_extension.h"
+#include "utarray.h"
+#include "utstring.h"
+
+PyObject *CommonError;
+
+/* Initialize pickle module. */
+
+PyObject *pickle_module = NULL;
+PyObject *pickle_loads = NULL;
+PyObject *pickle_dumps = NULL;
+PyObject *pickle_protocol = NULL;
+
+void init_pickle_module(void) {
```

Comment:
I think this module makes more sense as part of Ray. Maybe the only part that is needed in Plasma is `PyStringToUniqueID`, which would be fine to duplicate.